docs/source-pytorch/tuning/profiler_advanced.rst

```diff
@@ -72,3 +72,115 @@ Make sure the code is running while you are trying to capture the traces. It wil
+    ones_like              0.21%            3.461us          0.81%            13.669us         13.669us
+    sum_out                0.45%            7.638us          0.74%            12.432us         12.432us
+    transpose              0.23%            3.786us          0.68%            11.393us         11.393us
+    as_strided             0.60%            10.060us         0.60%            10.060us         3.353us
+    to                     0.18%            3.059us          0.44%            7.464us          7.464us
+    empty_like             0.14%            2.387us          0.41%            6.859us          6.859us
+    empty_strided          0.38%            6.351us          0.38%            6.351us          3.175us
+    fill_                  0.28%            4.782us          0.33%            5.566us          2.783us
+    expand                 0.20%            3.336us          0.28%            4.743us          4.743us
+    empty                  0.27%            4.456us          0.27%            4.456us          2.228us
+    copy_                  0.15%            2.526us          0.15%            2.526us          2.526us
+    broadcast_tensors      0.15%            2.492us          0.15%            2.492us          2.492us
+    size                   0.06%            0.967us          0.06%            0.967us          0.484us
+    is_complex             0.06%            0.961us          0.06%            0.961us          0.481us
+    stride                 0.03%            0.517us          0.03%            0.517us          0.517us
+    ---------------------  ---------------  ---------------  ---------------  ---------------  ---------------
+    Self CPU time total: 1.681ms
+
+.. note::
+    Since HPUProfiler extends PyTorch Profiler, when using the HPUProfiler, wall clock time will not be representative of the true wall clock time.
+    This is due to forcing profiled operations to be measured synchronously, when many HPU ops happen asynchronously.
+    It is recommended to use this Profiler to find bottlenecks/breakdowns, however for end to end wall clock time use
+    the ``SimpleProfiler``.
+
+----
+
+***************************
+Profile a distributed model
+***************************
+To profile a distributed model, use the :class:`~pytorch_lightning.profilers.hpu.HPUProfiler` with the *filename* argument which will save a report per rank.
+
+.. code-block:: python
+
+    from pytorch_lightning.profilers import HPUProfiler
+
+    profiler = HPUProfiler(filename="perf-logs")
+    trainer = Trainer(profiler=profiler)
+
+----
+
+*****************************
+Visualize profiled operations
+*****************************
+To visualize the profiled operations, enable **export_to_chrome** in the :class:`~pytorch_lightning.profilers.hpu.HPUProfiler` (Default: True).
+
+.. code-block:: python
+
+    from pytorch_lightning.profilers import HPUProfiler
+
+    profiler = HPUProfiler()
```

Comment:
I know it's a default value for the moment, but then the code block doesn't really show anything.

```suggestion
    profiler = HPUProfiler(export_to_chrome=True)
```